chia/data_layer/download_data.py

```diff
@@ -92,7 +92,8 @@ async def insert_into_data_store_from_file(
     store_id: bytes32,
     root_hash: Optional[bytes32],
     filename: Path,
-) -> None:
+) -> int:
+    num_inserted = 0
```

Comment:
This looks like debugging noise from the other stuff you were figuring out?  Seems reasonable to have, just making sure I'm not missing a relationship to the core change here.

Reply:
Yea, it's somewhat unrelated but I added some logging that reports how many nodes were inserted when we inserted this delta file. Probably of some limited value but I don't think hurts anything so inclined to leave it in